x-pack/plugins/cases/server/telemetry/schema.ts

```diff
@@ -26,6 +28,32 @@ const countSchema: CountSchema = {
   daily: long,
 };
 
+interface AttachmentRegistrySchema {
+  type: 'array';
+  items: AttachmentItemsSchema;
+}
+
+const attachmentRegistrySchema: AttachmentRegistrySchema = {
+  type: 'array',
+  items: {
+    average: long,
+    maxOnACase: long,
+    total: long,
+    type: string,
+  },
+};
+
+const attachmentFrameworkSchema: AttachmentFrameworkSchema = {
+  persistableAttachments: attachmentRegistrySchema,
+  externalAttachments: attachmentRegistrySchema,
+  files: {
+    average: long,
```

Comment:
Nit: do we want to keep the decimals in the avg? If so, we might need to mark it as double or float. WDYT?

Reply:
Great question, I was thinking integers since I don't know if care about the precision that much for this telemetry. I'll ask my PM though 👍

Reply:
I checked with my PM and we're going to keep them as longs.